### PR TITLE
Update docs for tf.linalg.triangular_solve

### PR DESCRIPTION
**Pull request by lc0**
Not sure if `api_def_MatrixTriangularSolve.pbtxt` is going to be rendered as I expect it to be rendered

As a part of TensorFlow docs sprints from Munich.

cc @dynamicwebpaige

### CODE REVIEW

**Comment by lc0** [comment]
any updates on this one? @dynamicwebpaige @annarev

File: tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt [hunk 1]
```diff
@@ -53,5 +52,34 @@ The output is a tensor of shape `[..., M, K]`. If `adjoint` is
...
+
+x = tf.linalg.triangular_solve(a, b, lower=True)
+x
+# <tf.Tensor: id=257, shape=(4, 1), dtype=float32, numpy=
+# array([[ 1.3333334 ],
+#        [-0.66666675],
+#        [ 2.6666665 ],
+#        [-1.3333331 ]], dtype=float32)>
+
+a@x
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt]: Does this work? It doesn't for me.

**Review by martinwicke — CHANGES_REQUESTED** [review]

File: tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt (hunk 1 above)
Comment (lc0) [reply tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt]: https://colab.research.google.com/drive/1e_7Xvmq1eHCcfL4W5lJn7vLRRUVfuWC1#scrollTo=BEvr6sT22nN_

do you mean, that for 1.13 I would need to enable eager? 
Just tested and it works for `1.13` as well as for `2.0.0-dev20190422`

Comment (lc0) [reply tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt]: ah, you use python 2.7, right? both of my envs are python3

Comment (lc0) [inline tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt]:
```suggestion
# in python3 one can use `a@x`
tf.matmul(a, x)
```

Comment (lc0) [reply tensorflow/core/api_def/base_api/api_def_MatrixTriangularSolve.pbtxt]: @martinwicke Thanks for review! Updated

**Review by martinwicke — APPROVED** [review]

**Pull request merged**